bookmarks/images/tests/test_views.py

```diff
@@ -0,0 +1,26 @@
+from django.test import TestCase
+from images.tests.test_modelmixin import ModelMixin
+from django.urls import reverse
+
+
+class TestDetailView(ModelMixin, TestCase):
+    def test_detail_view_returns_to_success_view(self):
+        self.client.login(**self.credentials)
+        self.client.post(
+            reverse("images:create"),
+            {
+                "title": "rehman",
+                "url": "https://upload.wikimedia.org/wikipedia/commons/thumb/5/50/Berlin_Opera_UdL_asv2018-05.jpg/800px-Berlin_Opera_UdL_asv2018-05.jpg",
+            },
+        )
+        response = self.client.get(
+            reverse("images:detail", args=[1, "rehman"])
```

Comment:
args 1 is implicit. Create image in the mixin setup method like below and use self.image.id instead of hard-coding one.

`self.image = Image.objects.create()`